### PR TITLE
Create 2017-08-28-react-native-monthly-3.md

### PR DESCRIPTION
**Pull request by grabbou**
Third meeting notes.

### CODE REVIEW

File: blog/2017-08-28-react-native-monthly-3.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- We plan to continue with the automated closing of stale issues for the foreseeable future. We’d like to be in a state where every impactful issue opened in the tracker is acted upon, but we’re not there yet. We need all the help we can from maintainers to triage issues and make sure we don't miss issues that introduce regressions or introduce breaking changes, especially those that affect newly created projects. People interested in helping out can use the Facebook GitHub Bot to triage issues and pull requests. The new maintainers guide at facebook.github.io/react-native/docs/maintainers.html contains more information on triage and use of the GitHub Bot. Please add yourself to the issue task force at https://github.com/facebook/react-native/blob/master/bots/IssueCommands.txt and encourage other active community members to do the same!
+
```
Comment (ssssssssssss) [inline blog/2017-08-28-react-native-monthly-3.md]: Shall FB update the status of the roadmap in monthly manner? :)

Comment (hramos) [reply blog/2017-08-28-react-native-monthly-3.md]: What do you mean?

**Comment by grabbou** [comment]
Thanks @hramos, I appreciate your language feedback!

**Comment by grabbou** [comment]
I have updated the description as per @tenodi hints on Slack. Please double check @hramos if you are okay with the wording and ship it. I am okay with it and don't have further adjustments to make.

File: blog/2017-08-28-react-native-monthly-3.md (hunk 1 above)
Comment (ssssssssssss) [reply blog/2017-08-28-react-native-monthly-3.md]: Dear @hramos An uptodate roadmap. https://github.com/facebook/react-native/wiki/Roadmap

**Comment by hramos** [comment]
Yup this is good to go. I'm currently working on getting Circle back to green to ensure the post gets published when this gets merged.

**Comment by hramos** [comment]
This could not be imported because internally, arcanist attempted to parse the `cc: @hramos @tenodi` line in the PR description and could not find any Phabricator users with the usernames "@hramos" or "@tenodi". 

I updated the description and will try again.

File: blog/2017-08-29-react-native-monthly-3.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- We have landed HMR support into [`haul`](https://github.com/callstack-io/haul) and a bunch of other, cool stuff! Check out latest releases.
+- React Native EU 2017 is coming! Next month is all about React Native and Poland! Make sure to grab last tickets available [here](https://react-native.eu/)
+
+### Expo
+
+- Released support for installing npm packages on [Snack](https://snack.expo.io). Usual Expo restrictions apply -- packages can't depend on custom native APIs that aren't already included in Expo. We are also working on supporting multiple files and uploading assets in Snack. [Satyajit](https://github.com/satya164) will talk about Snack at [React Native Europe](https://react-native.eu/).
+- Released SDK20 with camera, payments, secure storage, magnetometer, pause/resume fs downloads, and improved splash/loading screen.
+- Continuing to work with [Krzysztof](https://github.com/kmagiera/react-native-gesture-handler) on [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler). Please give it a try, rebuild some gesture that you have previously built using PanResponder or native gesture recognizers and let us know what issues you encounter.
```
[the remark this answers is not in the document]
Comment (hramos) [reply blog/2017-08-29-react-native-monthly-3.md]: Yup. Can you send a quick PR to fix? This already landed in master.

**Pull request closed without merging**